vmm/src/memory_manager.rs

```diff
@@ -262,6 +296,15 @@ impl MemoryManager {
             Ok(())
         })?;
 
+        if let Some(region) = virtiomem_region {
+            memory_manager.lock().unwrap().create_userspace_mapping(
```

Comment:
Allocate_mmio_addresses() may be called to reserve the mmio_address, incase that area of memory is used by MMIO later.

Reply:
Fixed.